bindata/v4.1.0/kube-controller-manager/leader-election-cluster-policy-controller-rolebinding.yaml

```diff
@@ -1,11 +1,19 @@
+# This rolebinding binds role for creation of leader lock configmap for upgrades 4.2-> 4.3
+# cluster-policy-controller is split from openshift-controller-manager in 4.3
+# leader lock in openshift-controller-manager NamespaceSecurityAllocationController and in ClusterPolicyController
+# cluster-policy-controller container runs in ns openshift-kube-controller-manager static pod
+# The locks, role, and rolebinding can be removed in 4.4
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   namespace: openshift-kube-controller-manager
-  name: system:openshift:leader-lock-cluster-policy-controller
+  name: system:openshift:lease-lock-cluster-policy-controller
 roleRef:
   kind: Role
-  name: system:openshift:leader-locking-cluster-policy-controller
+  name: system::lease-lock-cluster-policy-controller
 subjects:
 - kind: User
   name: system:kube-controller-manager
+- kind: ServiceAccount
```

Comment:
ugly, but I'll let you have it to get this thing crash landed.

Reply:
ok, fix w/ a follow-up